src/App.jsx

```diff
@@ -32,11 +32,9 @@ const App = () => {
     }
 
     if (restore && localStorage.getItem('loggedIn')) {
-      console.log('restoring session');
       session.login({
         oidcIssuer: SOLID_IDENTITY_PROVIDER,
-        redirectUrl,
-        onError: console.error
```

Comment:
We shouldn't leave console statements in production code. It pollutes the browser console, slows it down, and may accidentally expose implementation details that we want to keep hidden.